src/test/java/AlertAndMultiselect.java

```diff
@@ -0,0 +1,107 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class AlertAndMultiselect {
+    WebDriver driver;
+
+    @BeforeEach
+    public void openBrowser() {
+        driver = new ChromeDriver();
+    }
+
+    @Test
+    public void selectMultipelValues() {
+        driver.get("https://www.seleniumeasy.com/test/basic-select-dropdown-demo.html");
+
+        Select selectObject = new Select(driver.findElement(By.id("multi-select")));
+        selectObject.selectByVisibleText("Florida");
+        selectObject.selectByIndex(3);
+        selectObject.selectByValue("Washington");
+        WebElement loginButton = driver.findElement(By.cssSelector("#printAll"));
+        loginButton.click();
+
+        List<WebElement> selectedOptions = selectObject.getAllSelectedOptions();
+        List objects = new ArrayList();
+        for (WebElement selectedOption : selectedOptions)
+            objects.add(selectedOption.getText());
+
+        List<String> list = new ArrayList<>();
```

Comment:
Give normal name and ... = Arrays.asList("asdasd", "ssss"...);

Reply:
done